docs/.vuepress/theme/components/Page.vue

```diff
@@ -187,13 +198,20 @@ export default {
   },
   mounted() {
     this.selectedLang = localStorage?.getItem('selected_lang') ?? 'JavaScript';
+    this.themeClassName = this.getThemeClassName(localStorage?.getItem('handsontable/docs::color-scheme'));
```

Comment:
The LocalStorage API exists in all supported browsers.
```suggestion
    this.themeClassName = this.getThemeClassName(localStorage.getItem('handsontable/docs::color-scheme'));
```